### ISSUE TITLE
Feature Offer: Ability to "watch" a stock for periodic updates.

### ISSUE THREAD

**Issue by jspenc72** [aurbano/robinhood-node#23]
Any interest in adding a service to continually poll a set of stocks and use an event emitter on updates?

**Comment by aurbano** [comment]
This sounds like it could be useful, but probably more as an alert system, where the user can set specific thresholds and have an event fire when they are met. Comments on this?

**Comment by jspenc72** [comment]
I Have written a wrapper for this that adds the event functionality and added few other endpoints.

**Comment by jspenc72** [comment]
I am testing it noe

**Comment by jspenc72** [comment]
I am also adding a decision maker, comparison, and condition testing libraries. That way you can continually check the market conditions and trade accordingly.

**Comment by jspenc72** [comment]
Have you already written anything like that?

**Comment by jspenc72** [comment]
It could also be used for notifications.

**Comment by aurbano** [comment]
This sounds really good!
I haven't written anything like it so far but it sounds very exciting, let me know when there is anything I can start playing with!

**Comment by bvallelunga** [comment]
@jspenc72 I have interest in building something similar. Any updates on your progress?

**Comment by jspenc72** [comment]
It's built, I use it privately now to monitor and make trades.

**Comment by jondcallahan** [comment]
any chance you could open source that @jspenc72 ?

**Comment by busse** [comment]
Not to threadjack, but I've been working on some similar functionality here. It's very early stage, but feel free to check it out and open any Issues for features you'd find useful: https://github.com/busse/friartuck

**Comment by jspenc72** [comment]
@jondcallahan Sry for the extended silence. Additional endpoints with support for order cancellation have been merged into master. 

I am working on something that involves and Angular 2 dashboard, Observables and web sockets instead of events....

Once this is ready to go Ill open source what I can.

**Comment by aurbano** [comment]
I'll close this issue since this feature is not something that would make sense built into the client itself I believe.

**Comment by jspenc72** [comment]
@bvallelunga @jondcallahan @busse @aurbano 

Hey guys, per popular request Ive open sourced an nom module that implements RxJS for creating Obsvervables to simplify quotes and orders. Let me know what you think!

https://github.com/jspenc72/robinhood-observer
https://www.npmjs.com/package/robinhood-observer
